portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSingleSelectionField.testcase

```diff
@@ -32,13 +32,35 @@ definition {
 	}
 
 	@description = "Verify that a Single Selection Field can be created"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanBeCreated {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-132189 CanBeCreated pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Single Selection");
+
+		Click(locator1 = "Button#SAVE");
```

Comment:
Hi Evanilson,
We already have a macro created that saves the form.
Can you please replace the line 52 to `Form.save()` ?

You can find an example [here](https://github.com/liferay/liferay-portal/blob/f620c75a2a63893f3d86f675cd35bc75ef980f2b/portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase#L567), along with some others on this same file.